### ISSUE TITLE
Output path customization

### ISSUE THREAD

**Issue by volyx** [mike-neck/graalvm-native-image-plugin#20]
I run `graalvm-native-image-plugin` in Github Actions two times for MacOs and Linux. It would be nice to have an opportunity to store executable in different folders. Now first binary files overrides the second file.

Now I have such workaround:

```java
import org.gradle.internal.os.OperatingSystem

String osName = OperatingSystem.current().getNativePrefix()

task createOutputDir(type: DefaultTask) {
    File file = file("$buildDir/native-image/$osName")
    if (!file.exists()) {
        file.mkdirs();
    }
}

createOutputDir.dependsOn build

nativeImage {
    setGraalVmHome(System.getProperty("java.home"))
    setMainClass(mainClassName)
    setExecutableName("$project.name")
//    setOutputDirecory("$buildDir/native-image/$osName")
    arguments(
            "--no-fallback",
            "--enable-all-security-services",
            "--report-unsupported-elements-at-runtime",
            "-H:EnableURLProtocols=http",
            "-H:+AllowVMInspection",
            "-H:+ReportExceptionStackTraces",
            "-Dfile.encoding=UTF-8",
            "-H:Path=$buildDir/native-image/$osName"
    )
}
```

https://github.com/volyx/griffin/blob/bb3297fc12cfd8b80a606b026e75711fa32d631e/griffin/build.gradle#L99

**Comment by mike-neck** [comment]
Added `setOutputDirectory` method and the plugin generates a native binary under it.

https://github.com/mike-neck/graalvm-native-image-plugin/blob/bbb6453f359399047a2ff938212696a67fdc654d/src/functionalTest/resources/java-project/build-gradle-output-directory.txt#L14-L20

https://github.com/mike-neck/graalvm-native-image-plugin/blob/bbb6453f359399047a2ff938212696a67fdc654d/src/functionalTest/java/org/mikeneck/graalvm/GraalvmNativeImagePluginFunctionalTest.java#L57-L76

**Comment by volyx** [comment]
Cool! Thanks! 👍

**Comment by mike-neck** [comment]
The added method `setOutputDirectory` is available at the version `0.4.0` of this plugin.

https://plugins.gradle.org/plugin/org.mikeneck.graalvm-native-image
